src/test/java/org/xmlcml/cml/element/main/CMLBandListTest.java

```diff
@@ -0,0 +1,62 @@
+/**
+ *    Copyright 2011 Peter Murray-Rust et. al.
```

Comment:
Hi, the copyright statement format is as followed. First, in most countries the author or authors of a patch is/are the copyright holder. If you copied and started with code from someone else, that copyright is indeed often kept. But the copyright statement should also have the year of the new code (2023) and the name of the author. Your name, if you write the code all by yourself.

Please update the copyright statements. This is important to us to be compatible with projects like Debian, Red Hat, etc.